run-tests.php

```diff
@@ -3719,4 +3737,79 @@ function check_proc_open_function_exists()
     }
 }
 
+class OutputHandler
+{
+
+    /**
+     * @var null|string
+     */
+    private $htmlOutputFilename = null;
+
+    /**
+     * @var null|resource
+     */
+    private $htmlHandle = null;
+
+    /**
+     * @param string $outputFilename
+     */
+    public function setHtmlOutputFilename($outputFilename)
```

Comment:
Why is this not using a type declaration?
```suggestion
    public function setHtmlOutputFilename(string $outputFilename)
```

Reply:
See the conversation below. Making this incompatible with php 5.x isn't something we've discussed. Once we've had that dsicussion then I'll update this file, respectively.